### DISCUSSION TITLE
OGC SensorThingsAPI

### DISCUSSION THREAD

**Discussion by KoalaGeo** [TerriaJS/terriajs#5254] [Q&A] [answered]
Does Terriajs/terriamap support ingestion of SensorThingAPI data? Do the graphing functions work with this?

**Comment by KeyboardSounds** [comment, marked as answer]
Hey @KoalaGeo,

Unfortunately, no. We'd love to be able to support it though. If you or someone you know could put together a PR, we'd be thrilled it merge it.

**Reply by KoalaGeo** [reply]
I'd love to but sadly my technical skills are limited to spinning up an instance on k8s with some config changes :-(
